misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/ResultFeatureAssertionsSpec.kt

```diff
@@ -95,10 +105,14 @@ abstract class ResultFeatureAssertionsSpec(
                     messageContains("value: 1", "$toBeDescr: 2")
                 }
             }
-            //TODO #204
-            //it("${isFailureFeature.name} - throws AssertionError showing the expected type and the expected message") {
-            //    ...
-            //}
+            it("${isFailureFeature.name} - throws AssertionError showing the expected type and the expected message") {
+                expect {
+                    expect(resultSuccess).isFailureFun {}
```

Comment:
define something like:
```suggestion
                    expect(resultSuccess).isFailureFun { messageContains("hello") }
```